tests/units/common/test_enhancer.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from common.enhancer import Enhancer
+from freezegun import freeze_time
+
+
+class PseudoParsesOutput:
+    def get_output(self):
+        return {
+            "abstracts": "this is abstracts",
+            "copyright_holder": "copyright_holder",
+            "copyright_year": "2020",
+            "copyright_statement": "copyright_statement",
+            "copyright_material": "copyright_material",
+            "date_published": "2022-05-20",
+            "title": "title",
+            "subtitle": "subtitle",
+        }
+
+
+@pytest.fixture(scope="function")
+def generic_pseudo_parser_output():
+    return {
+        "abstracts": "this is abstracts",
+        "copyright_holder": "copyright_holder",
+        "copyright_year": "2020",
+        "copyright_statement": "copyright_statement",
+        "copyright_material": "copyright_material",
+        "date_published": "2022-05-20",
+        "title": "title",
+        "subtitle": "subtitle",
+    }
+
+
+publisher = "Springer"
+expected_output = {
+    "abstracts": [{"value": "this is abstracts", "source": publisher}],
+    "acquisition_source": {
+        "source": publisher,
+        "method": publisher,
+        "date": "2022-05-20T00:00:00",
+        "submission_number": "path/to/the/file",
+    },
+    "copyright": [
+        {
+            "holder": "copyright_holder",
+            "year": "2020",
+            "statement": "copyright_statement",
+            "material": "copyright_material",
+        }
+    ],
+    "imprints": [{"date": "2022-05-20", "publisher": publisher}],
+    "record_creation_date": "2022-05-20T00:00:00",
+    "titles": [{"title": "title", "subtitle": "subtitle", "source": publisher}],
+}
+
+
+@freeze_time("2022-05-20")
+def test_all_contructors(generic_pseudo_parser_output):
+    output = generic_pseudo_parser_output
+    enhancement = Enhancer()
+    enhanced = enhancement(item=output, publisher=publisher)
+    assert enhanced == expected_output
+
+
+@pytest.fixture(scope="function")
+def empty_generic_pseudo_parser_output():
+    return {
+        "abstracts": "",
+        "copyright_holder": "",
+        "copyright_year": "",
+        "copyright_statement": "",
+        "copyright_material": "",
+        "date_published": "",
+        "title": "",
+        "subtitle": "",
+    }
+
+
+expected_output_from_empty_input = {
```

Comment:
Same here. Put it as fixture or directly in the function as it seems to be used only once